### ISSUE TITLE
problems with connection to mgw-lan

### ISSUE THREAD

**Issue by Seqway** [grimmpp/home-assistant-eltako#113]
I followed the documentation but I do not know how to connect to my LAN Gateway. The doc says:

`eltako:
  gateway:
  - id: 1
    device_type: mgw-lan
    base_id: FF-80-80-00        # baseId of LAN gateway`

How do I find the base-id ?
the ID from my FAM14 is it not because it seems not to connect to it.

**Comment by grimmpp** [comment]
You can find out the base if with e.g. [eo_man](https://github.com/grimmpp/enocean-device-manager)

**In the config ip address is missing in field `address`.**

**Comment by Seqway** [comment]
> You can find out the base if with e.g. [eo_man](https://github.com/grimmpp/enocean-device-manager)
> 
> **In the config ip address is missing in field `address`.**

I dient postet it but it is in the config.yaml:

`eltako:
  gateway:
  - id: 1
    device_type: mgw-lan
    base_id: FF-A4-74-00      # baseId of LAN gateway
    address: 192.168.40.231     # address is only required for LAN gateway`

**Comment by grimmpp** [comment]
Do you have maybe any logs which let us know what is going wrong?

**Comment by Seqway** [comment]
> Do you have maybe any logs which let us know what is going wrong?

Yes I can send logs. But I need to know how to do it to send properly information to you. So where to look in HA and send it to you ?

**Comment by grimmpp** [comment]
I found something which I could fix. However I could not reproduce you issue. Can you test the new version? I already merged it into main but didn't create a new release.

**Comment by Seqway** [comment]
> I found something which I could fix. However I could not reproduce you issue. Can you test the new version? I already merged it into main but didn't create a new release.

Ok i installed the new version and with my port number I do get a connection BUT it is every 10 sec ON and the OFF. This is done every 10 sec.
Pretty similar to someone else - see [also forum](https://community.simon42.com/t/enocean-lan-gateway-integrieren/22677/49?u=mainzled)

**Comment by grimmpp** [comment]
My assumption is that your gateways either sends a different keep alive message or nothing. After 10sec the integration thinks there is a problem and tries to reconnect. 
You can now disable this function. by using `auto_reconnect: False` in the config. Better would be to add the keep alive message from your device.

**Comment by legnad** [comment]
I can confirm that setting `auto_reconnect: False` solves the 10 sec ON/OFF connection issue on my device (Piotek FAM-USB 515 via Ser2Net).

**Comment by grimmpp** [comment]
This is an old configuration question from 2024. Gateway setup and identity handling have since been redesigned and documented, while the current MGW-LAN transmission problem is tracked in #204. Closing this obsolete report for versions after 2.2.
